assets/aml-benchmark/components/batch_resource_manager/spec.yaml

```diff
@@ -4,7 +4,7 @@ is_deterministic: False
 name: batch_resource_manager
 display_name: Batch Inference Resource Manager
 description: Resource Manager for batch inference.
-version: 0.0.6
+version: 0.0.7
```

Comment:
can you please check if we might need to update batch_benchmark_config_generator, batch_inference_preparer, batch_output_formatter (as they are also using model evaluation environment).

Reply:
No, they use a different environment